### PR TITLE
[ciflow] remove ciflow/all as an option

### PR DESCRIPTION
**Pull request by suo**
Remove this from the list of supported labels, and provide a message
explaining why.

### CODE REVIEW

File: torchci/lib/bot/ciflowPushTrigger.ts [hunk 1]
```diff
@@ -146,13 +145,22 @@ async function handleLabelEvent(context: Context<"pull_request.labeled">) {
...
+    if (label === "ciflow/all") {
+      body =
+        "We have recently removed the `ciflow/all` label. People were using it because it sounded comprehensive, ";
+      body +=
+        "but it runs very expensive periodic CI jobs. If you just want to check that you won't be reverted, use `ciflow/trunk`. ";
+      body +=
+        "If you *really* want the old `ciflow/all` behavior, add `ciflow/trunk` and `ciflow/periodic`.";
+    } else {
+      body = `We have recently simplified the CIFlow labels and \`${label}\` is no longer in use.\n`;
+    }
```
Comment (malfet) [inline torchci/lib/bot/ciflowPushTrigger.ts:158]:
```suggestion
        "`ciflow/all` label has been recently removed, as it were used too frequently to imply that both PR and trunk tests should be run,  ";
      body +=
        "but it also runs a very expensive periodic CI jobs and binary builds. If you just want to check that you won't be reverted, use `ciflow/trunk`. ";
      body +=
        "If you *really* want the old `ciflow/all` behavior, add `ciflow/trunk` and `ciflow/periodic` and `ciflow/binaries`.";
    } else {
      body = `We have recently simplified the CIFlow labels and \`${label}\` is no longer in use.\n`;
    }
```

**Review by malfet — APPROVED** [review]
LGTM, but perhaps we can rephrase the message to avoid classifying contributors into "we" and "people"?

Also, not sure if `ciflow/all` also implied `ciflow/binaries`. If it were not, then this is one more reason to exclude it

**Review by atalman — APPROVED** [review]

File: torchci/lib/bot/ciflowPushTrigger.ts [hunk 2]
```diff
@@ -137,7 +137,6 @@ async function handleLabelEvent(context: Context<"pull_request.labeled">) {
   const valid_labels = [
     "ciflow/trunk",
     "ciflow/periodic",
-    "ciflow/all",
     "ciflow/android",
     "ciflow/binaries",
```
Comment (atalman) [inline torchci/lib/bot/ciflowPushTrigger.ts:141]: Looks like "ciflow/binaries" are kept so we should be ok, I thin we need this label.

**Comment by suo** [comment]
It did not run binaries, but otherwise the suggested verbiage is probably better :P

**Pull request merged**